tsconfig.json

```diff
@@ -37,6 +37,9 @@
     {
       "path": "./packages/dev-server-rollup/tsconfig.json"
     },
+    {
+      "path": "./packages/dev-server/tsconfig.json"
```

Comment:
These are shuffling around after I ran the generating tsconfig script. Not really a change.